src/shared_realm.hpp

```diff
@@ -127,6 +127,13 @@ class Realm : public std::enable_shared_from_this<Realm> {
     // functions which take a Schema from within the migration function.
     using MigrationFunction = std::function<void (SharedRealm old_realm, SharedRealm realm, Schema&)>;
 
+    // A callback function to be called the first time when a schema is created.
+    // It is passed a SharedRealm with in-transaction state at the version after schema
```

Comment:
It is passed a SharedRealm which is in a write transaction with the schema initialized.